test/driver.js

```diff
@@ -577,6 +577,13 @@ class Driver {
         return;
       }
 
+      if (task.noChrome && window?.chrome) {
+        this._log(`Skipping file "${task.file} (because on Chrome)"\n`);
```

Comment:
It looks like the second quotation mark should probably be moved a little bit.
```suggestion
        this._log(`Skipping file "${task.file}" (because on Chrome)\n`);
```